tabs/src/Wallet.tsx

```diff
@@ -11,13 +11,16 @@ const Wallet: React.FC = () => {
   );
   const [activePeriod, setActivePeriod] = useState(7); // Set default to 7-days
   const [showYourWalletTab, setshowYourWalletTab] = useState(true);
+  const [activeWalletTabName, setActiveWalletTabName] = useState<string>('Your wallet');
```

Comment:
`const [activeWalletTabName, setActiveWalletTabName] = useState<string>('Private');`

Reply:
Actually extra points if you use `WalletType` in `global.d.ts`:

```
type WalletType = 'Allowance' | 'Private';
```

Restricts code to only use those two bits of string.

Reply:
Keeping it to use for 'Private' and 'Allowance' for now